<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_ttlock</class>
 <widget class="QMainWindow" name="Login_ttlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sign_in_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User :</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>-20</y>
      <width>511</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x3:1, y2:2, stop:0 rgba(195, 233, 154), stop:1 rgba(1, 229, 161, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_login">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_up_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>sign_in_button</zorder>
   <zorder>login_label</zorder>
   <zorder>user_input</zorder>
   <zorder>password_input</zorder>
   <zorder>user_label</zorder>
   <zorder>password_label</zorder>
   <zorder>checkBox_login</zorder>
   <zorder>sign_up_button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
